data-model/annotations/schema/annotation.json

```diff
@@ -2,99 +2,207 @@
   "$schema": "http://json-schema.org/draft-2020-12/schema#",
   "type": "object",
   "properties": {
-    "ods:id": { "type": "string" },
-    "ods:type": { "const": "Annotation" },
-    "version": { "type": "integer" },
+    "ods:id": {
+      "type": "string"
+    },
+    "ods:type": {
+      "const": "Annotation"
+    },
+    "version": {
+      "type": "integer"
+    },
     "oa:motivation": {
-      "type": "string",
-      "enum": ["ods:adding", "oa:assessing", "oa:editing", "oa:commenting"]
+      "enum": [
+        "ods:adding",
+        "oa:assessing",
+        "oa:editing",
+        "oa:commenting"
+      ]
+    },
+    "oa:motivatedBy": {
+      "type": "string"
     },
-    "oa:motivatedBy": { "type": "string" },
     "oa:target": {
       "type": "object",
       "properties": {
-        "ods:id": { "type": "string" },
-        "ods:type": { "type": "string" },
-        "selector": {
+        "ods:id": {
+          "type": "string"
+        },
+        "ods:type": {
+          "type": "string"
+        },
+        "oa:selector": {
```

Comment:
this is correct. selector is from oa namespace https://www.w3.org/ns/anno.jsonld